src/App.tsx

```diff
@@ -1,4 +1,6 @@
+/* eslint-disable react/jsx-one-expression-per-line */
```

Comment:
Don't disable the eslint without extra need and make sure that the code follows this rule. It will improve the code readability